runtime/stateManager.py

```diff
@@ -112,6 +125,10 @@ def add_pipe(self, process_name, pipe):
         pipe.send(RUNTIME_CONFIG.PIPE_READY.value)
 
     def create_key(self, keys, send=True):
+        """
+        Updates contents in ROBOT: creates an entry from an absolute path
```

Comment:
This is too literal a description. Instead, it should be something like "Insert keys into state"